mission_1.css

```
@@ -0,0 +1,262 @@
+*{
+  box-sizing: border-box;
+  margin:0;
+  padding:0;
+}
+header{
+  width: 1920px;
+  height: 70px;
+  padding: 10px 200px 9px 200px;
+  gap: 0px;
```

0px인 경우 단위를 생략할 수 있답니다!
```suggestion
  gap: 0;
```